apiserver/facades/controller/instancepoller/instancepoller_test.go

```diff
@@ -968,7 +968,7 @@ func (s *InstancePollerSuite) TestSetProviderNetworkClaimProviderOrigin(c *gc.C)
 	dev := mocks.NewMockLinkLayerDevice(ctrl)
 	dExp := dev.EXPECT()
 	dExp.MACAddress().Return("00:00:00:00:00:00").MinTimes(1)
-	dExp.Name().Return("eth0")
+	dExp.Name().Return("eth0").MinTimes(1)
```

Comment:
Is this deterministic or we just don't care as long as it's called...?

Reply:
This isn't important for ensuring test correctness - it's just a statement of "this will be called, and whenever it is, return that value".